internal/manifest/report.go

```diff
@@ -27,6 +30,9 @@ type Report struct {
 	Root string
 	// Entries holds all reported content, indexed by their path.
 	Entries map[string]ReportEntry
+	// lastHardLinkID is used internally to allocate unique HardLinkID for hard
+	// links.
+	lastHardLinkID atomic.Uint64
```

Comment:
Why is this atomic when nothing else is concurrency-safe? (e.g. map right above)

Reply:
Removed in https://github.com/canonical/chisel/pull/179/commits/44149b7ca2d88a2f486ade4de5d4ff69b945c54f.